css/style.css

```diff
@@ -257,6 +257,77 @@ hr {
 }
 
 
+
+/******************issue.104*****************/
+ table {
```

Comment:
Don't change global styles use css id or class selector along with tags.

Reply:
Use colors from the existing color scheme.

Reply:
done

Reply:
Still using global styles use something link `.section table`.